lib/src/utils/date-fns-utils.js

```diff
@@ -0,0 +1,141 @@
+import addDays from 'date-fns/add_days';
+import addMonths from 'date-fns/add_months';
+import addYears from 'date-fns/add_years';
+import endOfDay from 'date-fns/end_of_day';
+import endOfMonth from 'date-fns/end_of_month';
+import endOfWeek from 'date-fns/end_of_week';
+import endOfYear from 'date-fns/end_of_year';
+import format from 'date-fns/format';
+import isAfter from 'date-fns/is_after';
+import isBefore from 'date-fns/is_before';
+import isSameDay from 'date-fns/is_same_day';
+import isValid from 'date-fns/is_valid';
+import setDay from 'date-fns/set_day';
+import setHours from 'date-fns/set_hours';
+import setMinutes from 'date-fns/set_minutes';
+import setYear from 'date-fns/set_year';
+import startOfDay from 'date-fns/start_of_day';
+import startOfMonth from 'date-fns/start_of_month';
+import startOfWeek from 'date-fns/start_of_week';
+import startOfYear from 'date-fns/start_of_year';
+import getHours from 'date-fns/get_hours';
+import getYear from 'date-fns/get_year';
+import isEqual from 'date-fns/is_equal';
+
+export default class DateFnsUtils {
```

Comment:
why using `class` here? Isn't good ol' object better for this purpose?

Reply:
1) Have an ability to extend this class and override some props (usefull for persian picker)
2) Easy to determine which utils are using via `typeof`

Reply:
That makes more sense, but both may be done with object too